app/live-steam-banner/liveStreamWidget.tsx

```diff
@@ -325,34 +281,30 @@ export const LiveStreamWidget = ({ isLive, event }: LiveStreamWidgetProps) => {
 
           <div className="bg-gray-75 px-4 py-2">
             <h3 className="mb-3 text-xl font-bold">About the Speaker</h3>
-            {!!speakersInfo.length &&
-              speakersInfo.map((speakerInfo, index) => (
+            {!!event?.presenterList.length &&
+              event.presenterList.map((presenter, index) => (
                 <div key={index} className="mb-8 grid grid-cols-6 gap-x-8">
                   <div className="col-span-1">
-                    {!!speakerInfo.PresenterProfileImage && (
+                    {!!presenter.presenter.profileImg && (
                       <Image
-                        src={speakerInfo.PresenterProfileImage?.Url}
-                        alt={speakerInfo.Title}
+                        src={presenter.presenter.profileImg}
+                        alt={presenter.presenter.presenter.name}
                         width={200}
                         height={200}
                       />
                     )}
                   </div>
                   <div className="col-span-5">
-                    <p className="mb-3 font-bold">{speakerInfo.Title}</p>
-                    <p
-                      dangerouslySetInnerHTML={{
-                        __html: sanitiseXSS(
-                          speakerInfo.PresenterShortDescription
-                        ),
-                      }}
-                    />
-                    {!!speakerInfo.PresenterProfileLink && (
+                    <p className="mb-3 font-bold">
+                      {presenter.presenter.presenter.name}
+                    </p>
+                    <TinaMarkdown content={presenter.presenter.about} />
+                    {!!presenter.presenter.presenter.peopleProfileURL && (
                       <CustomLink
                         className="float-right border-b-1 border-dotted border-gray-450 !no-underline"
-                        href={speakerInfo.PresenterProfileLink}
+                        href={presenter.presenter.presenter.peopleProfileURL}
                       >
-                        {`${speakerInfo.Title}'s profile&gt;`}
+                        {`${presenter.presenter.presenter.name}'s profile&gt;`}
```

Comment:
thats lots of presenters... Can we simplify that data structure?

if not maybe at the top you can do

```ts
const presenter = presenter.presenter;
```

Reply:
Done - did the second thing. Data structure + gql result structure is causing this repetition